etc/notebooks/test/test_plotting_libs.ipynb

```diff
@@ -222,7 +222,7 @@
    },
    "outputs": [],
    "source": [
-    "!pip install bokeh==0.11.1"
+    "!pip install bokeh"
```

Comment:
I would prefer if we keep this version scoped, to 0.12.x.  It's possible that a future update (such as 0.13) could break us again.